tracer/src/Datadog.Trace/IAST/IastUtils.cs

```diff
@@ -27,4 +26,14 @@ public static int GetHashCode(Array objects)
 
         return hash;
     }
+
+    public static int IdentityHashCode(object item)
+    {
+        return (item?.GetHashCode() ?? 0);
+    }
+
+    public static Range[] GetRangesForString(string stringValue, Source? source)
+    {
+        return new Range[] { new Range(0, stringValue.Length, source) };
+    }
```

Comment:
That would be nice to implement. I would like to add a new task in the backlog for this, as it would be an improvement that we could tackle. For simplicity, I would like to add this implementation in a separate PR. Thanks for the idea!